client/components/Trends/TrendsPage.vue

```diff
@@ -4,8 +4,9 @@
 <template>
   <main>
     <article v-if="$store.state.entries.length">
-      <div v-for="detail in new Set($store.state.entries.map(entry => entry.detail))" :key="detail">
-        <TrendsVisualization :entries="$store.state.entries.filter(entry => entry.detail === detail).reverse()" :detail="detail" />
+      <p>Click on the <b>Cognition</b>, <b>Pain</b>, or <b>Happiness</b> labels at the top of each chart to show/hide trend lines.</p>
+      <div v-for="detail in new Set(filteredEntries.map(entry => entry.detail))" :key="detail">
+        <TrendsVisualization :entries="filteredEntries.filter(entry => entry.detail === detail).reverse()" :detail="detail" />
         <hr />
       </div>
     </article>
```

Comment:
Should the `v-else` work for the tabs as well? Perhaps check for length of `filteredEntries` instead?

Reply:
this may have been my mistake when resolving the merge conflicts. I agree that it should be.